playground/frontend/playground_components/lib/src/widgets/output/output_tabs.dart

```diff
@@ -43,7 +43,7 @@ class OutputTabs extends StatelessWidget {
             playgroundController: playgroundController,
             name: 'widgets.output.result'.tr(),
             isSelected: tabController.index == 0,
-            value: playgroundController.codeRunner.outputResult,
+            value: playgroundController.codeRunner.resultLogOutput,
```

Comment:
I suggest renaming this parameter to something like `maxPossibleContent`. It took me awhile to figure out that it is used to check if we should add an update marker on the tab. This should be included in its doc comment.